Src/Test.UnitTests.Sarif.PatternMatcher.Sdk/ExtensionMethodTests.cs

```diff
@@ -129,5 +129,54 @@ public void ExtensionMethods_Anonymize()
 
             Assert.True(sb.Length == 0, sb.ToString());
         }
+
+        [Fact]
+        public void ExtenstionMethods_ContainUpperLowerAndDigitShouldBeAccurate()
+        {
+            string lowercaseOnly = "onlylowercase";
```

Comment:
>lowercaseOnly [](http://example.com/codeflow?start=19&length=13)

Nice work! This is a great opportunity to use a fluent assertionscope.

[https://timdeschryver.dev/blog/assertion-scopes-the-one-fluentassertions-feature-you-didnt-know-you-needed](https://timdeschryver.dev/blog/assertion-scopes-the-one-fluentassertions-feature-you-didnt-know-you-needed)